src/app/app-nav-bar/app-nav-bar.scss

```diff
@@ -44,7 +48,7 @@
   @include themify($themes) {
     color: themed('color-header') !important;
   }
-  font-size: 1.5rem;
+  font-size: 15px;
```

Comment:
Kan bli problem for UU som vi skal prøve å støtte på public-side. Ser ut som konsekvent bytter fra 'rem' til 'px' i hele løsningen? Vi mister da skalerbarhet i forhold til browser font-size som det er krav til i forhold til UU. Hvis jeg endrer font-størrelsen i browser påvirker ikke det lenger font-size på nettsiden. Forslag, fungerer det å bruke 'rem' som før, men at vi legger inn en "override" i fdk-theme på html-element som har font-size: 62.5%. Da vil 1rem være 10px med default font-size 16px i browser.

Reply:
Endret i fellesdatakatalog/theme package